trigger-service/README.md

```diff
@@ -0,0 +1,11 @@
+# Setting up a trigger
+
+A trigger can be created with the following command:
+
+```bash
+curl --request POST --data '{
+  "triggerName": "e0326da3de4b3d4ef4d193907afb82bf9afb938daccea445cbca747bb79c9139:NoOp:noOp",
+  "party": "alice",
+  "applicationId": "test-app-id"
+}' -H 'Content-Type: application/json' localhost:4002/v1/triggers
+```
```

Comment:
Should we rewrite the bootstrap script https://github.com/digital-asset/daml-platform-observability-example/blob/main/canton/bootstrap.canton at some point?

Reply:
Not sure if we can include this in the bootstrap. Honestly I added the example but more than a call that adds a point to the http metrics, we have no other metrics because the triggers themselves are not monitored yet.